src/main/webapp/quiz.html

```diff
@@ -1,49 +1,38 @@
 <!--gi Music: https://www.bensound.com -->
+ 
 <!DOCTYPE html>
 <html>
   <head>
     <meta charset="UTF-8">
     <title>Quiz Page</title>
-    <style src="style.css"></style>
+    <link rel="icon" href="/images/plantFav.png" type="image/x-icon">
+    <link rel="stylesheet" href="style.css">
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
-    <style>
-        #container {
-            margin-left: auto;
-            margin-right: auto;
-            width: 1500px;
-        }
-    </style>
     <audio controls id="background_music" style="display:none">
         <source src="/images/bensound-littleidea.mp3">
     </audio> 
   </head>
-  <body onload="checkSignIn()" onload="document.getElementById('background_music').play()">
-    <div id="container">
-        </br>
-        </br>
-        <h3 style="margin-left: 650px">Quiz Time!</h3>
-        </br>
-
-        <button type="button" class="btn btn-success btn-lg btn-block" id="start_quiz"> Start Quiz! </button>
-        <div id="quiz_time" style="display: none">
-            <div class="card">
-                <div class="card-body"> 
-                    <div id="quiz_question">
-                    </div>
-                </div>
-            </div>        
+ 
+  <body onload="document.getElementById('background_music').play()">
+    <div class="view">
+        <div id="container">
             </br>
-
-            <div class="card">
-                <div class="card-body"> 
-                    <div id="quiz_answers">
-                        <button id="submit_answer"> temporary </button>
-                    </div>
-                </div>
-            </div>
             </br>
+            <h3 id="headings">Quiz Time!</h3>
+            </br>
+ 
+            <button type="button" class="button" id="start_quiz"> Start Quiz! </button>
+            <div id="quiz_time" style="display: none">
+                <div id="questions">
+                </div>      
+                </br>
+                
+                <form action="/answer-quiz-question" method="POST" id="quiz_photos">
```

Comment:
what is this for? if u want to do it with the javascript, u dont need a form

Reply:
or does this get populated somewhere

Reply:
This gets populated to another servlet that will be in another pr. I didn't want to do the form in the js anymore